tests/Api/test_cases/protocols/input/rest-json.json

```diff
@@ -1154,6 +1154,53 @@
     ]
   },
   {
+    "description": "JSON Value Trait On Headers",
+    "metadata": {
+      "protocol": "rest-json",
+      "apiVersion": "2014-01-01"
+    },
+    "shapes": {
+      "InputShape": {
+        "type": "structure",
+        "members": {
+          "Foo": {
+            "shape": "StringType",
+            "location":"header",
+            "locationName": "Bar",
+            "jsonvalue": true
+          }
+        }
+      },
+      "StringType": {
+        "type": "string"
+      }
+    },
+    "cases": [
+      {
+        "given": {
+          "input": {
+            "shape": "InputShape"
+          },
+          "http": {
+            "method": "POST",
+            "requestUri": "/path"
+          },
+          "name": "OperationName"
+        },
+        "params": {
+          "Foo": "{\"a\":\"b\",\"c\":{\"d\":\"e\"}}"
```

Comment:
This doesn't look like the protocol test that was added to [other](https://github.com/aws/aws-sdk-js/blob/master/test/fixtures/protocol/input/rest-json.json#L1305) [SDKs](https://github.com/aws/aws-sdk-go/blob/cbdd5fa277e256b883fa15c9733f2134209ba0d0/models/protocol_tests/output/rest-json.json#L609).